admin-dev/themes/default/css/bundle/module/module.css

```diff
@@ -323,6 +346,15 @@ img.module-logo-thumb-list {
     margin-right: 5px;
 }
 
+.module-modal-body {
+    padding: 5px !important;
```

Comment:
"Does this padding makes me look fat ?"

With:
![capture du 2016-05-13 16-31-46](https://cloud.githubusercontent.com/assets/6768917/15251187/7c625b1a-1928-11e6-89a8-ee2e9934fee8.png)

Without:
![capture du 2016-05-13 16-35-33](https://cloud.githubusercontent.com/assets/6768917/15251264/d7d25f7c-1928-11e6-9d04-8bec74ee56d9.png)